js/app/modules/card/media.js

```diff
@@ -58,12 +59,15 @@ const Media = new Module.Class({
             attributions.push(this.model.copyright_holder);
         this.set_label_or_hide(this._attribution_label, attributions.map((s) => {
             return s.split('\n')[0];
-        }).join(' - ').toUpperCase());
+        }).join(' - '));
         this._attribution_button.visible = this._attribution_label.visible;
         this._attribution_button.connect('clicked', this._show_license_in_external_viewer.bind(this));
-
-        this._separator.visible = this._attribution_label.visible &&
-                                  this._caption_label.visible;
+        /* When both the caption & attribution are present in the lightbox,
+           add these classes so their spacing can be designed correctly. */
+        if (this._caption_label.visible && this._attribution_label.visible) {
```

Comment:
Posting again because my previous comment got eaten by an outdated diff:
The logic is not quite right here - this now assigns `--caption` and `--attribution` only if both are present. In order to get the benefit of having 2 classes, it should assign `--caption` if a caption is present, and `--attribution` if an attribution is present.